docs - update login troubleshooting (#22294)
metabase/metabase: docs/troubleshooting-guide/cant-log-in.md

- [I want to delete an account that was set up incorrectly][how-to-delete-an-account].

### I can't access the Metabase login page.

- [Are you using the right URL for your Metabase?][incorrect-metabase-url]
- [Has your Metabase account been deactivated?][deactivated-metabase-account]

#### Are you using the right URL for your Metabase?

1. Check whether you need to include a port number as well as a hostname in the connection URL. For example, Metabase might be at `https://example.com:3000/` instead of `https://example.com/`.
   - If you're an administrator, you'll have configured this.
   - If you're not, please ask your admin.
2. Check whether your Metabase instance has moved. For example, if you were using a trial instance of Metabase, but you're now in production, the URL might have changed.

#### Has your Metabase account been deactivated?

For obvious reasons, regular users can't reactivate deactivated accounts. If you're an administrator and you want to do this for someone else:

1. Go to **Admin Panel** and select **People**.
2. If no **Deactivated** tab is available, there are no deactivated accounts, so this isn't the problem.

> 3. If there _is_ a **Deactivated** tab, look for the user who isn't able to log in.

Reviewer: ```suggestion
3. If there _is_ a **Deactivated** tab, look for the person who isn't able to log in.
```

4. Click on the recycle loop arrow to reactivate the account.

## I can't log in to my Metabase Cloud account

Your Metabase store password is different from your Metabase Cloud admin password.

- [I forgot my Metabase store password][reset-store-password].
- [I forgot my Metabase Cloud password][how-to-reset-admin-password].

## SSO

- [Troubleshooting SAML][troubleshooting-saml]
- [SSO creates "unknown" users](https://github.com/metabase/metabase/issues/15484).

If you are an administrator, you can go to **Admin Panel** and select **People**, then search for a user and look for an icon beside their name. If they log in using Google credentials, Metabase displays a Google icon. If they log in using an email address and password stored in Metababse, no icon is shown. Note that the type of user is set when the account is first created: if you create a user in Metabase, but that person then logs in via Google or some other form of SSO, the latter's icon will _not_ show up next to their name.

> If you are an administrator and want to check SSO settings, go to **Admin Panel**, choose **Settings**, then select the **Authentication** tab. [This FAQ][auth] explains how to configure SSO for various providers.

Reviewer: ```suggestion
If you're an administrator and want to check SSO settings, go to **Admin Panel**, choose **Settings**, then select the **Authentication** tab.  To learn how to configure SSO for various providers, check out our [SSO page][auth].
```
